includes/class-sensei-question.php

```diff
@@ -770,6 +770,32 @@ public static function the_question_media( $question_id ) {
 		echo self::question_media_kses( self::get_the_question_media( $question_id ) );
 	}
 
+
+	/**
+	 * Return the answer feedback CSS classes (default and custom) based if the answer is correct or not
+	 *
+	 * @param [type] $question_id
+	 * @param [type] $answer_correct
+	 * @return array CSS classes
+	 */
+	private static function get_answer_feedback_classes( $question_id, bool $answer_correct ): array {
+		if ( $answer_correct ) {
+			$feedback_block = Sensei_Quiz::get_correct_answer_feedback_block( $question_id );
+			return [
+				'sensei-lms-question__answer-feedback--correct',
+				$feedback_block['attrs']['className'],
```

Comment:
A small issue here, if there is no custom classname provided, the 'className' index will be undefined -

![image](https://user-images.githubusercontent.com/6820724/180100445-eb2b3585-509d-422a-88fe-4ba64932a947.png)

I think we can do a check to see whether it exists or not before adding it to the array.